libs/SalesforceSDKCore/SalesforceSDKCore/SFUserAccountManagerNotificationsTests.m

```diff
@@ -0,0 +1,512 @@
+/*
+ Copyright (c) 2017-present, salesforce.com, inc. All rights reserved.
```

Comment:
You have to move it in the filesystem in `Finder`. The logical grouping in `Xcode` has no association with the physical location on the filesystem.